lib/cacheOptimization.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+var gltfPrimitiveToCesiumGeometry = require('./gltfPrimitiveToCesiumGeometry');
+var cesiumGeometryToGltfPrimitive = require('./cesiumGeometryToGltfPrimitive');
+var createAccessorUsageTables = require('./createAccessorUsageTables');
+var Cesium = require('cesium');
+var GeometryPipeline = Cesium.GeometryPipeline;
+var defined = Cesium.defined;
+
+module.exports = cacheOptimization;
+
+// Helper method to map accessor collections from the usageTable to an independent primitive (if there is one)
+function createIndicesToAttributeDictionary(gltf) {
+    var dictionary = {};
+    var meshes = gltf.meshes;
+    for (var meshId in meshes) {
```

Comment:
It is not; it is in the ao branch. I think forego using it for now since it is async and doesn't have a way to notify when it completes currently.